smart_open/tests/test_smart_open.py

```diff
@@ -193,16 +194,28 @@ def test_open_with_keywords_explicit_r(self):
         self.assertEqual(expected, actual)
 
     @unittest.skipIf(
-        pkgutil.find_loader('pathlib') is None,
-        "do not test pathlib support if pathlib is not available")
+        (pkgutil.find_loader('pathlib') is None and
+         pkgutil.find_loader('pathlib2') is None),
+        "do not test pathlib support if pathlib or backport are not available")
     def test_open_and_read_pathlib_path(self):
-        """If pathlib.Path is available we should be able to open and read."""
-        from pathlib import Path
+        """If ``pathlib.Path`` is available we should be able to open and read."""
+        fpath = os.path.join(CURR_DIR, 'test_data/cp852.tsv.txt')
+
+        # Import ``pathlib`` if the builtin ``pathlib`` or the backport
```

Comment:
I think we're duplicating unnecessary work here. We've already gone through the pain of importing the necessary pathlib module in smart_open_lib. So, we can just get rid of all this import stuff, and use:

```python
from smart_open.smart_open_lib import pathlib
path = pathlib.Path('/foo/bar')
```

whenever you need access to the actual pathlib module.